server/blob/blob-cassandra/src/test/java/org/apache/james/blob/cassandra/cache/CacheBlobStoreTest.java

```diff
@@ -139,6 +138,16 @@ public void shouldNotCacheWhenLowCost() {
         });
     }
 
+    @Test
+    public void shouldNotCacheWhenEmptyStream() {
```

Comment:
Just add a comment

```
// We did not succeeded to contribute caching of empty stream through PushBackInputStream content replay
// This limitation don't affect correctness and don't frighten the overall usefulness of the CachedBlobStore work.`

Reply:
Actually, it not limitation of PushBackInputStream.I think its more about technical, when we open a stream we dont know ( care the stream are empty or not), but when save to the cache, we only accept `byte[]`. Empty stream are differences with empty byte when save, that the problem.

Reply:
Mono.just(emptyByteArray) && Mono.empty are different then ;-)

Reply:
(I'll have a look)

Reply:
> Mono.just(emptyByteArray) && Mono.empty are different then ;-)

Yes, it leading to `empty byte array` in the cache and we cannot do anything about it ( or i dont know yet). So i think we should have a comment to right problem.

Reply:
Then, should we add comment or not?